ci/upload_app_distribution.sh

```diff
@@ -5,12 +5,14 @@ set -e
 # Buildkite uses a clean state for each step (for concurrency)
 source ./ci/prepare_env_buildkite.sh
 
+export FIREBASE_GROUP="internal-beta"
+
 # Buildkite branch equals to tag name if build was triggered by tag
 if [[ $BUILKITE_BRANCH  =~ ^v[0-9]+.* ]]; then
     export APP_VERSION_NAME=${BUILDKITE_BRANCH:1}
 fi
 
 echo "INFURA_API_KEY=$INFURA_API_KEY" > project_keys
 
-./gradlew assembleInternal assembleRinkeby assembleRelease
-./gradlew crashlyticsUploadDistributionInternal crashlyticsUploadDistributionRinkeby crashlyticsUploadDistributionRelease
+./gradlew assembleInternal
+./gradlew appDistributionUploadInternal
```

Comment:
Does this also require setup for app distribution in order to restore rinkeby and release? If that is the case maybe a TODO here too would be nice.